packages/react-magma-dom/src/components/Stepper/Step.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { CheckIcon, CrossIcon } from 'react-magma-icons';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+// import { I18nContext } from '../../i18n';
+
+/**
+ * @children required
+ */
+export interface StepProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
We can do something similar to how Mantine handles it here: https://github.com/mantinedev/mantine/blob/master/packages/@mantine/core/src/components/Stepper/Stepper.tsx#L208-#L224 where some of the props are private, or we can handle it with context. 

Public StepProps should be:
- label
- ariaLabel
- summary/description
- hasError
- testId
- isInverse

I don't think we need to ask for the stepStatus, as we can figure it out with logic:
if index === active index, status = current;
if index < active index, status = complete;
if index > active index, status = incomplete;